src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokenTestBase.cs

```diff
@@ -129,18 +145,15 @@ private static void GenerateSemanticEditBaseline(SemanticTokensFullOrDelta edits
                     {
                         builder.Append(i).Append(' ');
                     }
-                    builder.AppendLine("] ");
+                    builder.AppendLine("]");
                 }
-                builder.Append(edits.Delta.ResultId);
```

Comment:
Serializing ResultId was unnecessary and caused un-needed changes in files every time you re-generated the baseline.